### DISCUSSION TITLE
被訊息綁架的我們

### DISCUSSION THREAD

**Discussion by zoo100130** [1101-GS4719-C/Python-Community-GS4719-C#2243] [General]
發現現在幾乎用訊息對談的和在現實跟人交流的比例差不多，但其實想想這樣超不健康。
一方面傳訊息的時間成本，一來一回雙方讀訊息加回訊息的時間直接double再double，明明沒聊甚麼東西，一個小時就被吃掉了。
另一方面**對話幾乎只能用碎片化的語句來表達**，導致有時候在表達能力也下降了，包括寫文章、被叫上台回答問題，要回答的很通順變得有點困難。
BTW現在覺得那種每次都能回一大串的人根本超人🫡respect
ps.感覺socail media用越多，人就變得越抽象

![image](https://github.com/user-attachments/assets/2dc21d2d-cd5b-4f04-a7a0-cd9c50afd7b2)  /![image](https://github.com/user-attachments/assets/e1e1e5b0-24ed-41a8-9d69-055b578111bb)

**Comment by yuan3x2** [comment]
我覺得主要是看談話內容，如果是聊一些生活小事，就算讓我面對面也沒辦法生出長篇大論，但如果是聊一些議題，我認為就算是使用訊息也會很有意義。但通常聊天就是想放鬆一下，所以偶爾放鬆一下應該是可以被原諒的吧_(:3 」∠ )

**Reply by zoo100130** [reply]
yep, retweet!
我當然也享受跟朋友輕鬆的哈拉，但只是有感自己的思維這樣慢慢被碎片化，就像現在短影音看多了，會對再撰寫和面對長篇內容時感到不適應。

**Comment by Boying12** [comment]
我認為聊天對象很重要，因為選擇與誰交流會直接影響我們的心情和放鬆程度。訊息聊天主要也是想要尋找一種輕鬆的氛圍，這樣才能在忙碌的生活中獲得片刻的喘息。通常，我會選擇和朋友們互相哈拉，分享一些生活中的趣事或搞笑的經歷，這樣的對話不僅能讓我開心，也能增進彼此的情誼。相反地，如果對話的內容讓我感到有壓力，或者是涉及到一些沉重的話題，那麼我就不會積極回復，甚至可能會選擇沉默。這是因為我希望保持自己的心理健康，而不想讓負面的情緒影響到我的日常生活。對我來說，聊天是一種放鬆的方式，而不是一種負擔。因此，與能帶來正能量的朋友交流，才能讓我在繁忙的生活中感受到快樂和舒適。

**Comment by Chattenoire** [comment]
想要增強論述能力，打破訊息綁架，那就去讀有意義的書，這是唯一解方。

在讀書的過程中，為了理解，你必須用自己的文字說出一段話的意思，這時候你的論述就出現了；當然，你如果只是讀過一本書沒有掌握，那等於白讀。不過，讀書也要記得看自己有興趣的，畢竟只為了學習而看書的話，到後來會痛苦的。

至於不知道從哪讀起？也許可以看看敝人之前的貼文/回覆，挑個有興趣的領域。

**Comment by slps101023** [comment]
用訊息溝通雖然方便，但的確帶來一些隱藏的負面影響。碎片化的語句使得思考和表達逐漸變得「即時短暫」，而缺乏一個完整的溝通架構，長此以往可能會削弱我們的語言組織能力和專注力。
面對這種情況，也許可以設定一些時間段專門用來「不看訊息」，轉而專注於面對面的溝通或是長篇的閱讀和寫作，培養更連貫的思維方式。這樣不僅可以提升表達力，也有助於改善時間管理，讓時間被更高效利用。

**Comment by forestdew507** [comment]
如果有明確的對話重點或目標導向，覺得比起傳訊息直接通話會更有效、更真實，如果想要快速又傳送文字訊息，大推語音轉文字功能，雖然有時候會辨識錯誤，但基本上想傳遞的大方向不會有錯，收到訊息的人也都看得懂:)
TMI 有時候想回一大串還會怕對方收到訊息是不是內心 os : 這是訊息 ! ! !

**Comment by NiCL100** [comment]
我也覺得很多人都被訊息綁架了，吃飯時回訊息、走路時回訊息，連在追劇的時候也會不斷出現訊息通知。我覺得他們過的真累，沒辦法好好專心在一件事上。偶爾會問他們說：「你不會不喜歡像這樣要一直回訊息嗎？」，居然會有人回答說：「不會啊，過的蠻充實的。」；但當然也會有人無奈的說：「沒辦法啊。」總之，每個人都有自己的選擇，以及自己喜歡的事吧。

**Comment by qiu0109** [comment]
我覺得比起講電話，我更喜歡傳訊息，可能是有點社恐的關係，講電話的話會擔心有沒有冷場，但是傳訊息的話就可以放一陣子再回復。如果是重要的事情，傳訊息可以有比較多的時間思考訊息內容，並且可以留下紀錄，因此我覺得傳訊息是很棒的溝通方式。
然而只用訊息對話聊天也是有負面影響，只能大家自己去從中取得平衡。話說我覺得有很多訊息可以回的人很厲害，感覺他們應該很多朋友?

**Comment by wcyeee** [comment]
我也覺得主要要看對象和內容耶！我和我的同學也會在線上聊天，但不是同時在線上，通常都是有點像留言板的方式聊天，我覺得這個方式很不錯，可以慢慢想慢慢回，一次回一大串然後對方慢慢看慢慢回，都可以經過思考慢慢想，也可以試試看這種方法！
